apps/dashboard/config/configuration_singleton.rb

```diff
@@ -36,6 +36,14 @@ def ood_version
     @ood_version ||= (ood_version_from_env || version_from_file('/opt/ood') || version_from_git('/opt/ood') || "Unknown").strip
   end
 
+  def ood_bc_ssh_to_compute_node
+    if ENV['OOD_BC_SSH_TO_COMPUTE_NODE'] == nil
+      true
+    else
+      ENV['OOD_BC_SSH_TO_COMPUTE_NODE'].to_bool
```

Comment:
@KinanAlAttar, Morgan actually meant to use [`to_bool`](https://github.com/OSC/ondemand/pull/321/commits/1a9332635438e65d59154c48b25b977a87590bdd#diff-4cd79485ee8c2d5b7a7748532534ab76R259-R261) like the `load_external_config?` method.

This is the correct implementation:

```ruby
def ood_bc_ssh_to_compute_node
  to_bool(ENV['OOD_BC_SSH_TO_COMPUTE_NODE'] || true)
end
```

Reply:
Got it, I just committed the correct implementation.